src/application/CLI/AutoComplete.java

```diff
@@ -0,0 +1,150 @@
+package application.CLI;
+import org.jline.reader.*;
+import org.jline.terminal.Terminal;
+import org.jline.terminal.TerminalBuilder;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+public class AutoComplete {
+
+    private Terminal terminal;
+    private LineReader lineReader;
+
+    public AutoComplete() {
+        try {
+            this.terminal = TerminalBuilder.builder()
+                            .build();
+            /*this.lineReader = LineReaderBuilder.builder()
```

Comment:
Is there a reason this commented code is left behind?

Reply:
Nope I must have forgot to delete it.